libsrc/osic1p/cputc.s

```diff
@@ -36,10 +41,13 @@ L1:     cmp     #$0D            ; LF?
 cputdirect:
```

Comment:
> [...] I hope that this minimal speed impact is acceptable.

Surely! But the whole discussion isn't about speed but about size. Other targets intentionally omit scrolling to get a smaller `cputdirect`...